.grunt-config/webpack.js

```diff
@@ -136,14 +136,14 @@ const entry = {
 	// Temporary solution for the AI App in the Admin.
 	'ai-admin': path.resolve( __dirname, '../modules/ai/assets/js/admin/index.js' ),
 	'styleguide': path.resolve( __dirname, '../modules/styleguide/assets/js/styleguide.js' ),
-	'styleguide-app-initiator': path.resolve( __dirname, '../modules/styleguide/assets/js/styleguide-app-initiator.js' ),
```

Comment:
@ManorHazaz Do we have a test to ensure that this loads correctly on the frontend?

Reply:
We have playwrights E2E tests for the whole feature, it should be covered by them.

Reply:
@ManorHazaz Brilliant.